scss/input/_layout.scss

```diff
@@ -5,13 +5,16 @@
     $textbox-shadow: 0 2px 2px 1px rgba(0, 0, 0, .06) !default;
     $fieldset-margin: 30px;
 
+    $input-border-width: 1px !default;
+
     .k-textbox,
-    .k-input.k-textbox {
+    .k-input.k-textbox,
+    .k-textarea {
```

Comment:
I think it is more meaningful to use .k-textarea for textarea element. So I prefer keeping this class